### PR TITLE
Do not submit clinic if no ID

### PR DESCRIPTION
**Pull request by gsvr**

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps please review

File: test/clinic.test.js [hunk 1]
```diff
@@ -842,6 +842,55 @@ describe("app", function() {
                         .run();
                 });
             });
+
+            describe("if user did not provide an sa_id number", function() {
+                it.only("should not try to send jembi doc - send json only", function() {
```
Comment (imsickofmaps) [inline test/clinic.test.js]: hanging `.only`

File: test/fixtures.js [hunk 2]
```diff
@@ -400,6 +400,32 @@ module.exports = function() {
                 "swt": 1,
                 "dmsisdn": "+27001",
                 "cmsisdn": "+27001",
+                "id": "NI",
```
Comment (imsickofmaps) [inline test/fixtures.js]: think this should be `null`

**Comment by imsickofmaps** [comment]
@smn ready for review

File: go-app-chw.js [hunk 3]
```diff
@@ -176,7 +176,7 @@ go.utils = {
             return contact.extra.passport_no + '^^^' + contact.extra.passport_origin.toUpperCase() + '^FI';
           },
           'none': function () { // TODO - CHECK
```
Comment (smn) [inline go-app-chw.js]: remove TODO?

File: go-app-clinic.js [hunk 4]
```diff
@@ -1124,32 +1124,47 @@ go.app = function() {
...
+                                .then(function(json_result) {
+                                    var json_to_fire;
+                                    if (json_result.code >= 200 && json_result.code < 300){
+                                        json_to_fire = (([self.metric_prefix, "sum", "json_to_jembi_success"].join('.')));
+                                    } else {
+                                        json_to_fire = (([self.metric_prefix, "sum", "json_to_jembi_fail"].join('.')));
+                                    }
+                                    return self.im.metrics.fire.inc(json_to_fire, {amount: 1});
+                            });
+                        }
```
Comment (smn) [inline go-app-clinic.js]: any specific reason these functions are inline?

**Comment by smn** [comment]
Minor stuff :+1:

**Comment by gsvr** [comment]
good to go

**Comment by smn** [comment]
:+1:

**Pull request merged**